### DISCUSSION TITLE
PIP AS review op Mac

### DISCUSSION THREAD

**Discussion by RCDDA** [asreview/asreview#1905] [Troubleshooting (report errors in the "Issues" tracker)] [answered]
Hi all, for some reason, after install. Python, Pip is not found, cant seem to fix it have been trying for few weeks. Removed all softw. reinstallled Python,  Any advice would be a pleasure as pip is required prior to install ASreview for an syst. review. Kindly ask any guidance on tthe same. Greetings, Richard

**Comment by J535D165** [comment]
Can you check if pip is actually installed?

```
python3 -m pip --version
```

**Reply by RCDDA** [reply]
Thank you, yes it is installed. This is the response: "Last login: Wed Oct 23 22:33:21 on ttys000
richarddasselaar@MacBookAir ~ % python3 -m pip --version
pip 24.2 from /Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/pip (python 3.12)
richarddasselaar@MacBookAir ~ % "

**Comment by Rensvandeschoot** [comment]
Have you tried the solutions described in the documentation?? https://asreview.readthedocs.io/en/latest/troubleshooting.html

**Reply by RCDDA** [reply]
Hi Rens,

Thank you for your message. I hope you are well.

I tried the command: python -m pip install asreview, which did provide a response. May I ask if ASReview should now be active and working? its my intent to be carefull and thoughtfull.

Thank you so much for your time and thoughts.

P.S. We can talk in Dutch if you prefer.

Regards,
Richard Dasselaar

**Reply by Rensvandeschoot** [reply, marked as answer]
The next step is to start ASReview via the CLI using 

```
asreview lab
```

**Comment by RCDDA** [comment]
Thank you, this has been succesfuly executed. Much gratitude and a pleasure,
Honored to connect.
